pom.xml

```diff
@@ -63,7 +63,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-storage-parent</site.installationModule>
-    <google.core.version>1.91.3</google.core.version>
+    <google.core.version>1.92.3-SNAPSHOT</google.core.version>
```

Comment:
No snapshot dependencies.

Reply:
Doing it temporary until dependency PR is merged/released.